db/migrate/20160118115624_add_estimation_time_to_read.rb

```diff
@@ -0,0 +1,5 @@
+class AddEstimationTimeToRead < ActiveRecord::Migration
+  def change
+    add_column :links, :time_to_read, :string
```

Comment:
what if this article can take several hours to read or something like that? my main aim was to leave the ability to write any estimation